src/Task/ProtectedTaskManager.cpp

```diff
@@ -84,29 +86,50 @@ ProtectedTaskManager::IncrementActiveTaskPointArm(int offset)
 {
   ExclusiveLease lease(*this);
   TaskAdvance &advance = lease->SetTaskAdvance();
+  TaskWaypoint *twp = nullptr;
 
   switch (advance.GetState()) {
   case TaskAdvance::MANUAL:
-  case TaskAdvance::AUTO:
-    lease->IncrementActiveTaskPoint(offset);
+  case TaskAdvance::AUTO: {
+      lease->IncrementActiveTaskPoint(offset);
+      twp = lease->GetActiveTaskPoint();
+    }
```

Comment:
This scope is pointless - just remove the curly braces.
Are you aware that this code applies to both `AUTO` and `MANUAL`?

Reply:
Will remove the curly braces if you prefer.

It's a fallthrought and the compiler would flagged if it wasn't empty. I know.
Do you prefer not to use empty fallthroughs?

Reply:
Empty fallthroughs are ok, I was just wondering whether you were aware that this also covers `MANUAL`.